### DISCUSSION TITLE
Stuck on "fin sync"

### DISCUSSION THREAD

**Discussion by fnick851** [docksal/docksal#1755] [Q&A]
For a Drupal 9 project, when running "drush cim -y" locally, it gets stuck on the "Importing Drupal Configuration" message:
<img width="1699" alt="Screenshot 2023-02-10 at 14 13 53" src="https://user-images.githubusercontent.com/8741707/218178487-5a6ca468-e0b7-4eef-b735-9528192b11e5.png">

I believe it has issues to import the config. So I run "fin drush -v cim -y":
<img width="636" alt="Screenshot 2023-02-10 at 14 14 14" src="https://user-images.githubusercontent.com/8741707/218178740-9d9714e9-ccd7-4642-ba9f-631a21ce7ee5.png">
It shows these messages & gets stuck.

Could someone provide some ideas for how to fix this?

**Comment by MattDeckerIO** [comment]
@fnick851 `fin sync` is a custom command inside your commands directory specific to your installation: .docksal/commands/sync

The failure is occurring when Drupal attempts to import the configuration files. `drush cim -y`.

**Reply by fnick851** [reply]
Thanks @MattDeckerIO, lol
